docs/SIL.rst

```diff
@@ -5245,6 +5245,34 @@ independent of the operand. In terms of specific types:
 In ownership qualified functions, a ``copy_value`` produces a +1 value that must
 be consumed at most once along any path through the program.
 
+move_value
+``````````
+
+::
+
+   sil-instruction ::= 'move_value' sil-operand
+
+   %1 = move_value %0 : $@_moveOnly A
+
+Performs a move of the operand, ending its lifetime. When ownership is enabled,
+it always takes in an `@owned T` and produces a new `@owned @_moveOnly T`. 
```

Comment:
@gottesmm Both the operand and result of `move_value` may be either moveOnly or copyable. 
Whether either value is moveOnly depends on its type. The important thing about this operation is that it can do a type conversion in *either* direction between 'moveOnly T' and 'T'. It can also simply perform a move without doing any type conversion.

For the purpose of SIL ownership, it simply forwards an owned value. It has no special properties.